mvapi.yaml

```
@@ -232,6 +272,19 @@ components:
         name:
           type: string
           description: Unique but short name of the grade, like "Excellent" or "To reject".
+    Judge:
+      type: object
+      properties:
+        id:
+          type: string
+          readOnly: true
+        has_voted:
+          type: string
+        poll_id:
```

```suggestion
        has-voted:
          type: string
        poll-id:
```

Maybe should we follow same naming convention here?